bindings/csharp/http/batch/program.cs

```diff
@@ -30,18 +30,23 @@
 var builder = WebApplication.CreateBuilder(args);
 var app = builder.Build();
 
-if (app.Environment.IsDevelopment()) {app.UseDeveloperExceptionPage();}
+if (app.Environment.IsDevelopment()) { app.UseDeveloperExceptionPage(); }
 
 var httpClient = new HttpClient();
 httpClient.DefaultRequestHeaders.Accept.Add(new System.Net.Http.Headers.MediaTypeWithQualityHeaderValue("application/json"));
 
 // Triggered by Dapr input binding
-app.MapPost("/" + cronBindingName, async () => {
+app.MapPost("/" + cronBindingName, async () =>
+{
      Console.WriteLine("Processing batch..");
 
+     // Ensure the culture is set to one that uses dot notation
+     CultureInfo.CurrentCulture = CultureInfo.InvariantCulture;
```

Comment:
Isn't this better set one time outside the function handler?

Reply:
Yes you're right, but that requires implementing it inside the `WebApplicationBuilder`, which i did with the latest commit!